docs/design/coreclr/botr/corelib.md

```diff
@@ -82,7 +84,7 @@ Do not replicate the comments into your actual QCall implementation. This is for
 class Foo
 {
     // All QCalls should have the following DllImport attribute
```

Comment:
it sounds like CharSet and EntryPoint are mandatory. would it be more accurate to say:

```diff
- // All QCalls should have the following DllImport attribute
+ // All QCalls should have the following DllImport attribute (with or without `EntryPoint`)
or simply:
+ // An example of QCall DllImport
```

Reply:
CharSet and EntryPoint aren't mandatory, but the majority of cases will require at least one of them. We can update the docs/examples if necessary, but I think the guidance is good as a starting place.